providers/implementations/keymgmt/ec_kmgmt.c

```diff
@@ -161,6 +161,8 @@ int key_to_params(const EC_KEY *eckey, OSSL_PARAM_BLD *tmpl,
                 x = BN_CTX_get(bnctx);
             if (py != NULL)
                 y = BN_CTX_get(bnctx);
+            if (x == NULL && y == NULL)
+                goto err;
```

Comment:
It would be better to check the failures inside the ifs above instead of combining the check like this.

Reply:
Yes, I just noticed that and updated.